pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -8,6 +8,7 @@
 # Runtime
 , coreutils, faac, pciutils, procps, utillinux
 , pulseaudioSupport ? true, libpulseaudio ? null
+, alsaSupport ? stdenv.isLinux, alsaLib ? null
```

Comment:
I don't know if I like this default value. For most Linux users pulseaudioSupport is enough.

Reply:
@doronbehar That's not true. We have no `hardware.pulseaudio.enable = true;` by default. Also a lot of packages support bare ALSA setups by default, so zoom-us should too. Also makes no harm, since zoom sees both. Will use pulseaudio by default but fallback to alsa if pulseaudio is not running

Reply:
Oh right.. It won't increase the closure size as well I suppose.